docs.html

```diff
@@ -650,6 +652,27 @@ <h3>Javascript</h3>
         <iframe src="https://carbon.now.sh/embed/oXBQy0YXYfnjzQCiSAKp" sandbox="allow-scripts allow-same-origin">
         </iframe>
       </section>
+      <section id="rating">
+        <h1>Rating</h1>
+        <p>
+          Rating is powerful tool used for product reviews in review section.
+        </p>
+        <div class="example-container">
```

Comment:
![Screenshot (103)](https://user-images.githubusercontent.com/30971358/151747067-fccb0e7e-53d1-4b09-8b55-d7b49b8443ec.png)

Unable to see the rating component over here. Can you please check?

Reply:
![Screen Shot 2022-01-31 at 11 58 49 AM](https://user-images.githubusercontent.com/89216938/151748664-86d587ea-2933-4313-a058-170bf9ab7b16.png)

Reply:
hey raksha it's working fine in my local

Reply:
Maybe an issue with codesandbox? Not sure why this is happening.